emails/new_offerer_validation.py

```diff
@@ -0,0 +1,25 @@
+from pprint import pprint
+from typing import Dict
+
+from emails.beneficiary_booking_cancellation import SUPPORT_EMAIL_ADDRESS
+from models import Offerer
+from repository.feature_queries import feature_send_mail_to_users_enabled
+from repository.user_queries import find_all_emails_of_user_offerers_admins
+from utils.mailing import format_environment_for_email, DEV_EMAIL_ADDRESS
+
+
+def retrieve_data_for_new_offerer_validation_email(offerer: Offerer) -> Dict:
+    recipients = find_all_emails_of_user_offerers_admins(offerer.id)
```

Comment:
```suggestion
    recipients = offerer.user.email
```
Vu tout ce qu'on s'est dit tout à l'heure, tu ne devrais plus avoir besoin d'utiliser cette fonction.